.github/scripts/sync-github-releases.sh

```diff
@@ -198,7 +211,9 @@ _upload_artifacts() {
     _upload "otp_doc_man_${stripped_name}.tar.gz" "application/gzip"
     _upload "otp_win32_${stripped_name}.exe" "application/x-msdownload"
     _upload "otp_win64_${stripped_name}.exe" "application/x-msdownload"
-    }
+    _upload "otp_win32_${stripped_name}.zip" "win32"
+    _upload "otp_win64_${stripped_name}.zip" "win64"
```

Comment:
```suggestion
    _upload "otp_win32_${stripped_name}.zip" "application/zip"
    _upload "otp_win64_${stripped_name}.zip" "application/zip"
```

Reply:
that is actually on purpose as I use $2 in _upload to specify the "target" to pass to the workflow.

Reply:
Yikes, sorry about that, good thing I "asked" 😅